### ISSUE TITLE
Docformatter can break an RST cross-reference

### ISSUE THREAD

**Issue by mhsmith** [PyCQA/docformatter#124]
I see there are some existing issues about Sphinx and RST (e.g. #68, #60, #78, #61), but I don't see one mentioning this specific problem.

Docformatter version 1.5.0.

Input:
```
    def quadratic_curve_to(self, cpx, cpy, x, y):
        """Constructs and returns a :class:`QuadraticCurveTo <QuadraticCurveTo>`.

        Second line
        """
```
First pass (still OK):
```
    def quadratic_curve_to(self, cpx, cpy, x, y):
        """Constructs and returns a :class:`QuadraticCurveTo
        <QuadraticCurveTo>`.

        Second line
        """
```
Second pass (broken):
```
    def quadratic_curve_to(self, cpx, cpy, x, y):
        """Constructs and returns a :class:`QuadraticCurveTo.

        <QuadraticCurveTo>`.

        Second line
        """

**Comment by weibullguy** [comment]
This will be fixed by #61 as the requirement is to not break RST cross-references.

**Comment by kdeldycke** [comment]
I can confirm the issue, which not affect rST cross-references, but virtually any space-separated rST directive.

See for example this Python code:
```python
def setup():
    """Register ``..click:example::`` and ``.. click:run::`` directives, augmented with ANSI coloring."""
    pass
```

Which is formatted with a split in the middle of a double-backticked sequence:
```diff
--- before/./rst-docstring.py
+++ after/./rst-docstring.py
@@ -1,3 +1,6 @@
 def setup():
-    """Register ``..click:example::`` and ``.. click:run::`` directives, augmented with ANSI coloring."""
+    """Register ``..click:example::`` and ``..
+
+    click:run::`` directives, augmented with ANSI coloring.
+    """
     pass
```

These results have been produced with the latest development version of `docformatter` (d3696139b8ac61a6483975457e2811fb14d5b71c at the time of writing).

**Comment by kdeldycke** [comment]
This issue might be related to #79, in which a double invokation of `docformatter` produces an extra blank line of a long line, like in the result above.

**Comment by kdeldycke** [comment]
There's probably some inspiration on how to handle this in the recent PR that fixed URL wrapping (#115).

You can see there a special case for URLs using an rST syntax (`.. a link: https://example.com/index.html`): https://github.com/PyCQA/docformatter/commit/9c18ed433ffa85617a23df94079a08f882f754b4#diff-9a6d834c8022d3c6f558a340e5f2c78bab7adb15a880272d63f8b5d16d646a7cR938-R943

**Comment by fmigneault** [comment]
I'm forced to pin `docformatter==1.5.0` because of this new bug in `1.5.1`.
